dev/custom-pages/home.scss

```diff
@@ -0,0 +1,166 @@
+@import '../custom-base';
+
+.bg-gray {
+  background-color: #efefef;
+}
+
+.home-degree-search-wrap {
+  background-color: #fff
+}
+
+@media (min-width:992px) {
+  .home-degree-search-wrap {
+    background: -webkit-gradient(linear, left top, right top, from(#fff), color-stop(50%, #fff), color-stop(50%, #efefef), to(#efefef));
+    background: linear-gradient(to right, #fff, #fff 50%, #efefef 50%, #efefef 100%)
+  }
+}
+
+@media (min-width:992px) {
+  .home-degree-search-row {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-wrap: nowrap;
+    flex-wrap: nowrap
+  }
+}
+
+.home-degree-search-pad {
+  display: none
```

Comment:
d-none Athena class could have been used here

Reply:
I think in this case - which was taken from the other place we're using this style on the main site - it makes sense to define this here, instead of applying another class to the element in the markup or even including the already defined `d-none` class to this class. This way, all the rules that effect this empty div in the markup are defined in one place and easily understood.